bokeh/io/export.py

```diff
@@ -241,6 +241,14 @@ def wait_until_render_complete(driver):
     from selenium.webdriver.support.ui import WebDriverWait
     from selenium.common.exceptions import TimeoutException
 
+    def is_bokeh_loaded(driver):
+        return driver.execute_script('''
+            const b = window.Bokeh;
+            return b && b.documents && b.documents.length > 0;
+        ''')
+
+    WebDriverWait(driver, 5, poll_frequency=0.1).until(is_bokeh_loaded)
```

Comment:
Should this be in a `try/except` block, that converts the timeout exception into a `RuntimeError` with a more Bokeh-specific message?